apps/admin/secrets-csi-driver/secrets-csi-driver-gitrepo.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: source.toolkit.fluxcd.io/v1beta1
-kind: HelmRepository
+kind: GitRepository
```

Comment:
this is actually a helm repository: https://github.com/Azure/secrets-store-csi-driver-provider-azure/tree/master/charts

Reply:
think i just needed to update the url https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/main/charts/secrets-store-csi-driver/README.md